structure_tests/structure_test.go

```diff
@@ -34,59 +35,59 @@ func TestRunCommand(t *testing.T) {
 		stdout := outbuf.String()
 		stderr := errbuf.String()
 
-		for _, errStr := range tt.ExpectedError {
+		for _, errStr := range tt.GetExpectedError() {
```

Comment:
Did you try this out where the test types would know how to run themselves? That way it's a little more future proof in case things like the command types themselves change.